src/protocols/secure_channel/Constants.h

```diff
@@ -71,6 +71,8 @@ enum class MsgType : uint8_t
     CASE_Sigma2Resume = 0x33,
 
     StatusReport = 0x40,
+
+    ICD_CheckIn = 0X50,
```

Comment:
Suggestion was not applied correctly.....
```suggestion
    ICD_CheckIn = 0x50,
```